ui/src/app/workflows/components/workflow-filters/workflow-filters.tsx

```diff
@@ -57,13 +90,45 @@ export function WorkflowFilters(props: WorkflowFilterProps) {
         return results;
     }, [props.workflows, props.phaseItems]);
 
+    const handleNameFilterChange = (item: {title: string; id: string}) => {
```

Comment:
please prefer named functions as they are better for debugging/tracing, as per #12062 et al (and the non-memoized functions in this same component)

```suggestion
    function handleNameFilterChange(item: {title: string, id: string}) {
```

Reply:
this helper will also no longer be needed after below comment